index.js

```diff
@@ -63,6 +63,11 @@ export const SMSCalculator = {
     
     if (encoding === this.encoding.GSM_7BIT_EX) {
       length += this.getEscapedCharCount(text);
+    } else if (encoding === this.encoding.UTF16) {
+      length = [...text].reduce(
+        (acc, char) => acc + (/[\u0100-\uFFFF]/.test(char) ? 2 : 1),
```

Comment:
For consistency, I would add the regex to the following object:

![Screenshot 2024-06-24 at 11 40 14](https://github.com/fasttrack-solutions/sms-calculator/assets/6594254/53cc3c2e-d225-4cc6-b0a1-2e89b924c859)

And use a self-explanatory method name so we will remember what it does in the future. E.G: `getNonLatinCharCount()`